isobus/include/isobus/isobus/can_stack_logger.hpp

```diff
@@ -172,4 +176,45 @@ namespace isobus
 	};
 } // namespace isobus
 
+//! @cond Doxygen_Suppress
+#ifdef DISABLE_CAN_STACK_LOGGER
+/// @brief A macro which removes a "critical" log statement depending on the state of DISABLE_CAN_STACK_LOGGER
+/// @param logString A log statement
+#define LOG_CRITICAL(logString)
+/// @brief A macro which removes a "error" log statement depending on the state of DISABLE_CAN_STACK_LOGGER
+/// @param logString A log statement
+#define LOG_ERROR(logString)
+/// @brief A macro which removes a "warning" log statement depending on the state of DISABLE_CAN_STACK_LOGGER
+/// @param logString A log statement
+#define LOG_WARNING(logString)
```

Comment:
![image](https://github.com/Open-Agriculture/AgIsoStack-plus-plus/assets/29043784/7b6a15b4-6a9b-4d2d-9cd2-46cbbb9dd49a)
I get the error now that "macro "LOG_WARNING" passed 4 arguments, but takes just 1" when I disable the can stack logger. I think adding the `...` here as well will fix that